app/views/PageParents.vue

```diff
@@ -75,5 +553,561 @@
   }
 </script>
 
-<style>
+<style scoped>
```

Comment:
is it worth importing variables, and using those instead of hex codes?
i.e. https://github.com/codecombat/ozaria/blob/729343acc8453518254a65f983f704c96da2b90f/ozaria/site/styles/common/variables.scss#L1

Also understand we gotta go fast.

Reply:
Is there a coco version of this?  I wanted to use vars for this or pre made components but couldn't find it.  If there is I'll include in the next set of changes later this week.

Reply:
My mistake. I context switched to Ozaria whilst reviewing this. It may still be worth adding the variables even without that specific file.